src/adt_simple.erl

```diff
@@ -86,31 +86,26 @@ server(S = #state { gossip = GS }) ->
 						)
 					);
 				% New block and confirmed txs callback.
-				{NewGS, {new_block, _, _, B, _, _}} ->
-					ar:d({adt, recvd_new_block}),
-					FullTXs =
-						lists:map(
-							fun(T) -> ar_storage:read_tx(T) end,
-							B#block.txs
+				{NewGS, {new_block, _, _, BSD, _, _}} ->
```

Comment:
In some other places we are following the convention `B` - block shadow, `FullB` - full block.

Reply:
Changed it back in 87cd4a02
I find `BSD` more explicit though, let's chat about it